src/NBitcoin/Consensus.cs

```diff
@@ -23,7 +23,7 @@ public class Consensus : IConsensus
         public Money ProofOfStakeReward { get; }
 
         /// <inheritdoc />
-        public uint MaxReorgLength { get; }
+        public uint MaxReorgLength { get; private set; }
```

Comment:
Isn't `private set;` redundant in latest c#?

Reply:
Why do you need this change at all? (are you reflection changing MaxReorgLength ?)

Reply:
even if it is redundant, it is better to see it explicitly